src/modules/db_redis/redis_connection.c

```diff
@@ -137,14 +140,23 @@ int db_redis_connect(km_redis_con_t *con) {
     char hosts[MAX_URL_LENGTH];
     char* host_begin;
     char* host_end;
+    redisSSLContext *ssl = NULL;
     LM_DBG("connecting to redis cluster at %.*s\n", con->id->url.len, con->id->url.s);
     host_begin = strstr(con->id->url.s, "redis://");
     if (host_begin) {
         host_begin += 8;
-    } else {
-        LM_ERR("invalid url scheme\n");
+    } 
+    
+    if (db_redis_opt_ssl != 0) {
+        /* Create SSL context*/
+	redisInitOpenSSL();
+	ssl = redisCreateSSLContext(NULL, NULL, NULL, NULL, NULL, NULL);
+	if (ssl == NULL) {
+	    LM_ERR("Unable to create Redis SSL Context.\n");
```

Comment:
It seems indention here is wrong, please adapt it for the goto and the one bracket.